code/game/objects/structures/window.dm

```diff
@@ -444,7 +444,7 @@
 /obj/structure/window/plasma/reinforced
 	name = "reinforced plasma window"
 	desc = "A window made out of a plasma-silicate alloy and a rod matrice. It looks hopelessly tough to break and is most likely nigh fireproof."
-	icon_state = "rplasmawindow"
+	icon_state = "plasmawindow"
```

Comment:
Ok.... i don't even think I coded in the windoor. The sprite is in. also, with that change, it kinda looks like you're changing the reinforced plasma window texture to be a normal plasma window...?